cli/cmdlineparser.cpp

```diff
@@ -513,9 +505,16 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
 
             // print all possible error messages..
             else if (std::strcmp(argv[i], "--errorlist") == 0) {
-                mShowErrorMessages = true;
-                mSettings.xml = true;
```

Comment:
This was unnecessary and might have interfered with other options since we proceed with "left-to-right" rpocessing.